sbc_compassion/models/import_letters_history.py

```diff
@@ -271,9 +271,15 @@ def _analyze_attachment(self, file_, filename):
         # now try to find the layout
         # loop over all the patterns in the pattern directory
         template, pattern_center = self._find_template(img)
-        lang_id = self._find_language(img, pattern_center, template)
-
-        if self.force_template:
+        if template != self.env.ref('sbc_compassion.default_template'):
+            lang_id = self._find_language(img, pattern_center, template)
+            if len(lang_id) == 1:
+                lang_id = lang_id[0].id
```

Comment:
If you use only id of language, why not changing method `_find_language` in order to return list of ids instead of list of records ?
Furthermore, you can rename `_find_language` to `_find_languages`.
It would as well be good to rename variable `supporter_languages_id` of `import.letter.line` to `letter_language_id` which makes more sense.